### PR TITLE
Complete database week1

### PR DESCRIPTION
**Pull request by Merhawi-gebreslasie**

### CODE REVIEW

File: databases/week1/hyf_lesson1_homework_queries.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+USE hyf_lesson1;
+--1
+SELECT COUNT(*) AS 'count task '
```
Comment (dinythomas89) [inline databases/week1/hyf_lesson1_homework_queries.sql]: `*` is a much general approach. Preferably count(id) is more suited  and id's cannot be null

File: databases/week1/hyf_lesson1_homework_queries.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+--1
+SELECT COUNT(*) AS 'count task '
+ FROM task;
+ --2
+SELECT  COUNT(*) AS 'count task '
+ FROM task
+ WHERE due_date IS NULL;
+ --3
+ SELECT * FROM task t JOIN status s ON t.status_id=s.id
+ WHERE s.name ='Done';
```
Comment (dinythomas89) [inline databases/week1/hyf_lesson1_homework_queries.sql:12]: Looks more structured 
```
SELECT * 
FROM task t 
JOIN status s ON t.status_id=s.id
WHERE s.name ='Done';
```

**Review by dinythomas89 — APPROVED** [review]
Check this link about comments in mysql https://dev.mysql.com/doc/refman/8.0/en/comments.html#:~:text=MySQL%20Server%20supports%20three%20comment,newline%2C%20and%20so%20on).

**Comment by Merhawi-gebreslasie** [comment]
Thanks 👍

ons. 9. feb. 2022 21.23 skrev Diny Thomas ***@***.***>:

File: databases/week1/hyf_lesson1_homework_queries.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ SELECT * FROM task 
+ ORDER BY due_date DESC;
+ -- 6
+ SELECT * FROM task 
+ ORDER BY due_date DESC
+ LIMIT 1;
+ -- 7
+ SELECT title,due_date 
+ FROM task 
+ WHERE title REGEXP 'database'
```
Comment (martamatos) [inline databases/week1/hyf_lesson1_homework_queries.sql:27]: nice that you're using REGEXP :)

File: databases/week1/hyf_lesson1_homework_queries.sql [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+       OR description REGEXP 'detabase';
+--8
+
+SELECT task.title,status.name AS text 
+FROM task 
+JOIN status;
```
Comment (martamatos) [inline databases/week1/hyf_lesson1_homework_queries.sql:33]: join status on what?

**Pull request merged**